### ISSUE TITLE
IOPS usage metrics per VM(or how to extract types.PerfEntityMetric)

### ISSUE THREAD

**Issue by sofixa** [vmware/govmomi#472]
Hello.

I am trying to get statistics(mainly IOPS, because everything else i could need i already have with govc) about all the VMs, but i am a complete noob in Go.

I've inspired myself from this Gist https://gist.github.com/codekoala/1a10ed7071a8e50dfcc0, but when trying to run it, i get a thing like:

`2016/03/17 17:30:57 Response: (*types.PerfEntityMetric) &{PerfEntityMetricBase:{DynamicData:{} Entity:{Type:HostSystem Value:host-48}} SampleInfo:[{DynamicData:{} Timestamp:2016-03-17 16:30:40 +0000 UTC Interval:20}] Value:[0xc82017f320]`

But i have so far failed to extract the actual value from Value:[0xc82017f320] in an int form, so that i could use it. 

So, in short, my problem is that i don't know how to extract an int from types.PerfEntityMetric.

Thanks in advance to anybody who'd be kind enough to help me.

**Comment by cblomart** [comment]
In the gist take care: the printed value is only the first element of a slice containing *types.PerfEntityMetric.
vSphere performance datamodel is sometimes no so straightforward... (just look at their api doc)

In my own code to retrive managedobject perfs to graphtite (you can look at the code https://github.com/cblomart/vsphere-graphite) here how i get Int values.

here "base" is one element of returnval:

pem := base.(*types.PerfEntityMetric)
for _, genericserie := range pem.Value {
      serie := baseserie.(*types.PerfMetricIntSeries)
      log.Printf("Response: (%T) %+v", serie, serie)
}

Which should show you a bunch of slice of int64
